### PR TITLE
Added "new shell terminal needed" notice

### PR DESCRIPTION
**Pull request by franciscop**
Otherwise you get a weird error that seems totally unrelated and has no intuitive solution:

```
Error running CLI: failed to run cargo: No such file or directory (os error 2)
```

When the simple solution was to just reopen the shell/terminal.

### CODE REVIEW

File: docs/guides/getting-started/prerequisites.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ successful, the following line will appear:
 Rust is installed now. Great!
 ```
 
+:::note
+You now need to open a new shell for rust to work properly.
```
Comment (lorenzolewis) [inline docs/guides/getting-started/prerequisites.md]:
```suggestion
You may need to open a new shell for Rust to work properly after being installed.
```

@JonasKruckenberg any notes on this?

Comment (FabianLars) [reply docs/guides/getting-started/prerequisites.md]: 1) If we need to mention it we need to mention it for all platforms.
2) Why is rustup's warning not enough 😢

Comment (franciscop) [reply docs/guides/getting-started/prerequisites.md]: 1. I don't know how the shell works in other platforms, please feel free to add it if needed as well.
2. The warning I got was very mysterious and looked unrelated, hence the warning here, IIRC it was: "Error running CLI: failed to run cargo: No such file or directory (os error 2)"

Comment (FabianLars) [reply docs/guides/getting-started/prerequisites.md]: That's not the error i meant. When you install [rustup](https://rustup.rs/) it asks you to restart the terminal too 🤷

Comment (franciscop) [reply docs/guides/getting-started/prerequisites.md]: Oh sorry, I didn't see any relevant message about restarting the terminal ([nor I see any in the source code here?](https://sh.rustup.rs/)), what does the rustup warning say? If it's there I've probably have missed it. IIRC when the installation finished I just saw the message "Rust is installed now. Great!" and then assumed I was done with the pre-requisites and moved on, but then the cargo error came up, Google was very unhelpful and it took a while until I could figure out that I just had to restart the terminal.

Comment (lorenzolewis) [reply docs/guides/getting-started/prerequisites.md]: For reference, this is what you get after installing rustup

Screenshot 2022-06-23 at 10 46 54

Comment (lorenzolewis) [reply docs/guides/getting-started/prerequisites.md]: I'm going to close this one out, I think it was just a blip. If we see it causing an issue for more people then we can add it in. Thanks a bunch anyway!

Comment (franciscop) [reply docs/guides/getting-started/prerequisites.md]: Sure! I probably missed that somehow, thanks for the follow ups

**Pull request closed without merging**